staging/src/k8s.io/apiserver/pkg/storage/etcd/metrics/metrics.go

```diff
@@ -59,6 +59,13 @@ var (
 		},
 		[]string{"operation", "type"},
 	)
+	objectCounts = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "etcd_object_counts",
+			Help: "Current number of stored objects split by kind.",
```

Comment:
It's not "Current". It's number of objects from some period of time in the last minute (or whetever user will say). May be worth being explicit...

Reply:
I will change it, but frankly no metric is ever "current" in the strict sense:)